### PR TITLE
docs: update estimated release of the PR #1338

### PR DESCRIPTION
**Pull request by realdimas**
According to CHANGELOG.md [notes](https://github.com/thanos-io/thanos/pull/1338/commits/8ab5a881c320df3dc1dd224806b0f80577ba0db8#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR62) PR #1338 fixing issue #1337 was expected to be released as part of the `v0.6.1`.
This PR updates CHANGELOG.md to reflect the new estimate.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ The other `type` you can use is `JAEGER` now. The `config` keys and values are J
 
 - [#1284](https://github.com/thanos-io/thanos/pull/1284) Add support for multiple label-sets in Info gRPC service.
 This deprecates the single `Labels` slice of the `InfoResponse`, in a future release backward compatible handling for the single set of Labels will be removed. Upgrading to v0.6.0 or higher is advised.
-*breaking* If you run have duplicate queries in your Querier configuration with hierarchical federation of multiple Queries this PR makes Thanos Querier to detect this case and block all duplicates. Refer to 0.6.1 which at least allows for single replica to work.
+*breaking* If you run have duplicate queries in your Querier configuration with hierarchical federation of multiple Queries this PR makes Thanos Querier to detect this case and block all duplicates. Refer to 0.6.2 which at least allows for single replica to work.
```
Comment (bwplotka) [inline CHANGELOG.md]: This is quite vague info TBH. And yes, I wrote this initially ;p 
Let's maybe rephrase it:

```suggestion
*breaking* If you have duplicated `storeAPIs` in your Querier configuration with the hierarchical federation of the multiple Queries this PR makes Thanos Querier to detect this case and block all duplicates. Refer to `master` or future `v0.7.0` builds which allows for a single replica to work in this case.
```

.. as it's not planned to fetch the fix to 0.6.x. If that's necessary let us know. If it would be it should be done while ago. v0.7.0 first rc release is planned as described here:  https://thanos.io/release-process.md/

Comment (realdimas) [reply CHANGELOG.md]: Thanks, pulled in the recommendation.

**Comment by realdimas** [comment]
Yes, we are affected.
Currently we fan out StoreAPI requests to HA pair without any client-side or server-side load-balancing (it's not yet feasible in our setup).
We can discuss the setup in person in 6 days :-)

**Comment by bwplotka** [comment]
Have you tried master build? (:

**Comment by realdimas** [comment]
We didn't, we were burned by tag in the past a couple of times with unexpected regressions, so, trying to be more careful with upgrades now.
